### PR TITLE
[release/1.3.0] Move to Azure DevOps for PRs

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+jobs:
+- job: Ubuntu1604_Debug
+  displayName: Ubuntu 16.04 Debug
+  pool:
+    vmImage: ubuntu-16.04
+  steps:
+  - checkout: self
+    clean: true
+  - script: ./eng/common/cibuild.sh --configuration Debug --prepareMachine
+  - task: CopyFiles@2
```
Comment (chcosta) [inline azure-pipelines.yml]: nit: I don't think this CopyFiles step is needed.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  displayName: Ubuntu 16.04 Release
+  pool:
+    vmImage: ubuntu-16.04
+  steps:
+  - checkout: self
+    clean: true
+  - script: ./eng/common/cibuild.sh --configuration Release --prepareMachine
+  - task: CopyFiles@2
+    inputs:
+      contents: '**/artifacts/Release/**'
```
Comment (chcosta) [inline azure-pipelines.yml]: nit: remove the prefix `**/`.  I'm not even sure if that works.

**Comment by mmitche** [comment]
@tmat Jenkins CI got disabled a while back (webhook is missing) and the errors in the logs indicate that something got broken in the repo (it's trying to push symbols in the Pr build).  Want me to disable CI altogether or do you want to deal with the failures?

**Comment by tmat** [comment]
You can disable the CI

**Pull request closed without merging**